Locker/Helpers/BundleHelpers.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  BundleHelpers.swift
+//  Locker
+//
+//  Created by Zvonimir Medak on 20.10.2021..
+//  Copyright © 2021 Infinum. All rights reserved.
+//
+
+import Foundation
+
+class BundleHelpers {
+
+    public static var bundleResource: Bundle? {
+        let myBundle = Bundle(for: Self.self)
+        guard let resourceBundleURL = myBundle.url(forResource: "Locker", withExtension: "bundle"),
+              let resourceBundle = Bundle(url: resourceBundleURL) else { return nil }
+        return resourceBundle
+    }
+
+    private static var decoder: JSONDecoder {
+        let decoder = JSONDecoder()
+        decoder.keyDecodingStrategy = .convertFromSnakeCase
+        return decoder
+    }
+
+    public static func write(_ data: Data, to url: URL) {
+        // we'll have periodic updates to the library so we don't want to throw errors
+        do {
+            let decodedResponse = try BundleHelpers.decoder.decode(DeviceResponse.self, from: data)
+            let encodedData = try? JSONEncoder().encode(decodedResponse)
+            try encodedData!.write(to: url)
+        } catch {
+        }
```

Comment:
In that case, we should use optional try :) 
```suggestion
        // we'll have periodic updates to the library so we don't want to throw errors
        guard let decodedResponse = try? BundleHelpers.decoder.decode(DeviceResponse.self, from: data),
              let encodedData = try? JSONEncoder().encode(decodedResponse)
        else { return }
        try? encodedData.write(to: url)
```

Reply:
Oh and please avoid using force unwrap unless you are 100% certain that you'll have data on the time when method/property is called/used :)